src/com/github/igotyou/FactoryMod/FactoryModManager.java

```diff
@@ -29,13 +29,17 @@
 	private HashSet<Material> possibleInteractionBlock;
 	private Material factoryInteractionMaterial;
 	private boolean citadelEnabled;
+	private int redstonePowerOn;
+	private int redstoneRecipeChange;
 	private String compactLore;
 
 	public FactoryModManager(FactoryModPlugin plugin,
-			Material factoryInteractionMaterial, boolean citadelEnabled) {
+			Material factoryInteractionMaterial, boolean citadelEnabled, int redstonePowerOn, int redstoneRecipeChange) {
```

Comment:
@Maxopoly Further thoughts/discussion here? Sitting down to make changes/ fixes now.